generic/tests/netperf.py

```diff
@@ -74,13 +74,35 @@ def mtu_set(mtu):
         if netdst in br_in_use:
             ifaces_in_use = host_bridges.list_iface()
             target_ifaces = list(ifaces_in_use + br_in_use)
-        if params.get("netdst_nic1") in process.system_output(
-                "ovs-vsctl list-br", ignore_status=True, shell=True).decode():
-            ovs_list = "ovs-vsctl list-ports %s" % params["netdst_nic1"]
-            ovs_port = process.system_output(ovs_list,
-                                             shell=True).decode().splitlines()
-            target_ifaces = target_ifaces + \
-                params.objects("netdst_nic1") + ovs_port
+        if process.run("which ovs-vsctl && "
+                       "systemctl status openvswitch.service",
+                       ignore_status=False, shell=True):
+            ovs_br = netperf_base.ssh_cmd(host, "ovs-vsctl list-br")
+            if ovs_br:
+                for nic in vm.virtnet:
+                    if nic.netdst in ovs_br:
+                        ovs_br = nic.netdst
```

Comment:
`ovs_br` will be changed dynamically, so if there have multiple NICs, the second one cannot have a correct check `if nic.netdst in ovs_br`, because ovs_br became a br name, not a br list.

Reply:
Actually, the existing test involves setting up two network cards, but only one of the network cards has a backend that belongs to ovs or vdpa, while the other has a backend that is a bridge. Therefore, the determined OVS bridge name meets the test requirements.

Reply:
That's your setting and can't be taken for granted that everyone uses it that way.
The project is upstream and the users' environment can be anything.

Reply:
I have updated, please help review.